web/documents/AOFiles/gabaritAO.csv

```diff
@@ -1 +1 @@
-Alias;RdfType;ExperimentURI;Geometry;Species;Variety;ExperimentModalities;Repetition
+Alias;Geometry;ExperimentURI;Species;Variety;ExperimentModalities;Repetition
```

Comment:
remove this update. You can aslo remove the gabaritAO.csv file which is not used now that we have change the agronomical objects.